app/cmd/server.go

```diff
@@ -82,8 +88,14 @@ func ServerCmd() cli.Command {
 			},
 		},
 		Action: func(c *cli.Context) {
+			dataEngine := c.String("dataEngine")
+			if dataEngine != types.DataEngineTypeV1 && dataEngine != types.DataEngineTypeV2 {
+				logrus.Fatalf("Invalid data engine value: %s. Must be 'v1' or 'v2'", dataEngine)
+				dataEngine = types.DataEngineTypeV1
+			}
```

Comment:
If the dataEngine is `""`, can we set it to v1 data engine.

Reply:
Probably not because we do not know the data engine type of the volume.
Return an error instead.